sormas-ui/src/main/java/de/symeda/sormas/ui/samples/PathogenTestController.java

```diff
@@ -374,11 +378,15 @@ private void handleAssociatedEventParticipant(
 						showChangeAssociatedSampleResultDialog(dto, null);
 					}
 				} else {
-					showConvertEventParticipantToCaseDialog(eventParticipant, dto.getTestedDisease(), caseCreated -> {
-						if (eventDisease == null) {
-							handleCaseCreationFromContactOrEventParticipant(caseCreated, dto);
-						}
-					});
+					List<CaseDataDto> duplicatedCases =
+						FacadeProvider.getCaseFacade().getDuplicatesWithPathogenTest(eventParticipant.getPerson().getUuid(), dto);
```

Comment:
After refactoring, convert the `EventParticipantDto`'s person to a reference using `toReference()`.